frontend/src/modules/history/components/Translation.js

```diff
@@ -274,19 +274,33 @@ export class TranslationBase extends React.Component<InternalProps, State> {
 
                             { (!canComment && commentCount === 0) ? null :
                                 <Localized
-                                    id='history-Translation--button-comment'
+                                    id={ commentCount === 0 ?
+                                        'history-Translation--button-comment' :
+                                        'history-Translation--button-comments'
+                                    }
```

Comment:
It would be more readable if each `<button>` would be wrapped in their own `<Localized>` element.

Reply:
I tried that first but couldn't get it working. I'll have another go to see if I can work it out as I agree it would be much easier to understand what is happening there.